examples/common/maths.adb

```diff
@@ -206,6 +204,45 @@ package body Maths is
     end Scaling_Matrix;
 
     --  ------------------------------------------------------------------------
+
+    function To_3 (Array_4 : Single_Array) return Single_Array is
```

Comment:
OK.
Thanks for your comments.
These seemed to do I wanted but, in light of your comments, I'll remove them and use your comments to hopefully do a better job.